src/core/renderers/DroppableControl.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * ---------------------------------------------------------------------
+ * Copyright (c) 2020 EclipseSource Munich
+ * Licensed under MIT
+ * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
+ * ---------------------------------------------------------------------
+ */
+
+import { ControlProps, rankWith, uiTypeIs } from '@jsonforms/core';
+import {
+  ResolvedJsonFormsDispatch,
+  withJsonFormsControlProps,
+} from '@jsonforms/react';
+import React from 'react';
+
+import { EditorElement } from '../../editor/components/EditorElement';
+
+interface DroppableControlProps extends ControlProps {}
+const DroppableControl: React.FC<DroppableControlProps> = ({
+  uischema,
+  schema,
+  path,
+  renderers,
+  cells,
+}) => {
+  return (
+    <EditorElement wrappedElement={uischema}>
+      <ResolvedJsonFormsDispatch
+        uischema={uischema}
+        schema={schema}
+        path={path}
+        renderers={renderers?.filter(
+          (r) => r.renderer !== DroppableControlRenderer
+        )}
+        cells={cells}
+      />
+    </EditorElement>
+  );
+};
+const DroppableControlRenderer = withJsonFormsControlProps(DroppableControl);
```

Comment:
We should not wrap in the `withJsonFormsControlProps` for now. The wrapper modifies the props we get so some of the forwarded to renderers get confused as they expected different props. For example the object renderer will fail with the adjusted props.